src/requestforattestation/RequestForAttestation.spec.ts

```diff
@@ -115,6 +124,54 @@ describe('RequestForAttestation', () => {
     expect(request.claim.owner).toBeUndefined()
   })
 
+  it('compresses and decompresses the request for attestation object', () => {
+    const legitimationAttestationBob: Attestation = Attestation.fromRequestAndPublicIdentity(
+      legitimationRequest,
+      identityBob
+    )
+    const legitimationBob: AttestedClaim = AttestedClaim.fromRequestAndAttestation(
+      legitimationRequest,
+      legitimationAttestationBob
+    )
+    const reqForAtt: RequestForAttestation = buildRequestForAttestation(
+      identityBob,
+      'ctype',
+      {
+        a: 'a',
+        b: 'b',
+        c: 'c',
+      },
+      [legitimationCharlie, legitimationBob]
+    )
+
+    const sortedCompressedReqForAtt: CompressedRequestForAttestation = [
```

Comment:
Doesn't need to state, that it is "sorted"

Reply:
Updated.